src/status_im/contexts/chat/messenger/menus/pinned_messages/view.cljs

```diff
@@ -83,4 +77,11 @@
          :key-fn      list-key-fn
          :separator   [quo/separator {:style {:margin-vertical 8}}]}]
        [empty-pinned-messages-state
-        {:community? (boolean community)}])]))
+        {:community? (boolean community)
+         :theme      theme}])]))
+
+(defn- internal-pinned-messages
+  [params]
+  [:f> f-pinned-messages params])
```

Comment:
you shouldn't need to make this a functional component to work with `with-theme` wrapper btw - the with-theme method should handle this already 🤔